util/system/path_windows.go

```diff
@@ -26,10 +26,22 @@ func CheckSystemDriveAndRemoveDriveLetter(path string) (string, error) {
 		return "", errors.Errorf("No relative path specified in %q", path)
 	}
 	if !filepath.IsAbs(path) || len(path) < 2 {
-		return filepath.FromSlash(path), nil
+		return filepath.FromSlash(filepath.Clean(path)), nil
 	}
 	if string(path[1]) == ":" && !strings.EqualFold(string(path[0]), "c") {
 		return "", errors.New("The specified path is not on the system drive (C:)")
 	}
-	return filepath.FromSlash(path[2:]), nil
+	return filepath.FromSlash(filepath.Clean(path[2:])), nil
+}
+
+func IsAbs(pth string) bool {
```

Comment:
Can we have unit tests?

Reply:
Added some tests for ```IsAbs()``` for Windows. On Linux his function is just a wrapper for ```filepath.IsAbs()```. Please take another look when you have the chance.

Thanks!